src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -9,6 +9,7 @@
 
     /* Volunteer Prefix definitions */
     public static final Prefix PREFIX_NAME = new Prefix("n/");
+    public static final Prefix PREFIX_ID = new Prefix("i/");
```

Comment:
Maybe a prefix of 
'ic/' or 'nric/' would be more intuitive compared to just 'i/'?